### PR TITLE
Set close button size

### PR DESCRIPTION
**Pull request by pbkompasz**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [X] Refactoring

## Context

- [X] Part of #6988
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [X] My contribution builds **clean without any errors or warnings**
- [X] I've merged recent default branch -- **main** and I've no conflicts
- [X] I've tried to respect high code quality standards
- [X] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [X] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=16hqk8Tjugo9AwSzKkxXc1dsRK7iJetpx1YBaoXYwdHzuNAt&usdamount=50&donation=true)

#### Community participation

- [ ] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [X] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screenshot from 2023-08-28 11-39-25
Screenshot from 2023-08-28 11-39-17

### <samp>🤖 Generated by Copilot at c3718fd</samp>

Added a new class and a new prop type to the `NeoButton` component to enable more customization and functionality, and used them to improve the appearance and behavior of the close button in the shopping cart modal.

### <samp>🤖 Generated by Copilot at c3718fd</samp>

> _Oh, we're the coders of the sea, and we work on the UI_
> _We add some classes and some props to make the buttons look nice_
> _We heave and ho on the `is-align-items-center`_
> _And we sing a merry tune as we close the modal with `is-close-btn`_

### CODE REVIEW

**Comment by exezbcz** [comment]
Could you please apply this and the style to modals as well, its still a close button afaik 
image

thanks a lot!

File: libs/ui/src/components/NeoButton/NeoButton.scss [hunk 1]
```diff
@@ -216,3 +216,8 @@
 .is-fixed-width {
   width: 10rem;
 }
+
+.is-close-btn {
+  width: 24px;
+  height: 24px;
+}
```
Comment (roiLeo) [inline libs/ui/src/components/NeoButton/NeoButton.scss:223]: I would use [spacing helpers](https://bulma.io/documentation/helpers/spacing-helpers/) directly on button class

Comment (pbkompasz) [reply libs/ui/src/components/NeoButton/NeoButton.scss:223]: Should I add padding to the button based on the size of the icon?

**Comment by thnaylor** [comment]
I wonder if it would be beneficial to have a generic solution of a button with a icon that can be reusable since there is some inconsistency when it comes to using just icons as buttons.

My thoughts are a NeoButton with a "no-border" prop similar to the "no-shadow" implementation.

Examples:

with no-border prop: (default button padding)
image

multiple:
image

with text variant prop: (no padding)
image

🤷‍♂️

File: libs/ui/src/components/NeoButton/NeoButton.scss (hunk 1 above)
Comment (pbkompasz) [reply libs/ui/src/components/NeoButton/NeoButton.scss]: Screenshot from 2023-08-29 14-14-27
Screenshot from 2023-08-29 14-14-22
Screenshot from 2023-08-29 14-14-18

File: libs/ui/src/components/NeoButton/NeoButton.vue [hunk 2]
```diff
@@ -5,6 +5,9 @@
       'is-fixed-width': fixedWidth,
       'no-shadow': noShadow,
       'loading-with-label': loadingWithLabel,
+      'p-2': size === 'small' && isCloseBtn,
+      'p-1': size === 'medium' && isCloseBtn,
+      'px-1': size === 'large' && isCloseBtn,
```
Comment (roiLeo) [inline libs/ui/src/components/NeoButton/NeoButton.vue:10]: what's this???

**Comment by yangwao** [comment]
what about now @exezbcz 
image

**Comment by exezbcz** [comment]
> with text variant prop: (has no padding

if the cross has bigger clickable area and its not affected by the padding of the top nav (changing height) I think its better

**Review by roiLeo — COMMENTED** [review]
Any news on these? I would create a new component instead of overloading `NeoButton` component

**Comment by thnaylor** [comment]
@roiLeo I think this can be closed.

Please refer to https://github.com/kodadot/nft-gallery/pull/7080

**Comment by prury** [comment]
Closing this one as @thnaylor made new PR for it
